apps/src/aiTutor/views/AITutorChatWorkspace.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import style from './ai-tutor.module.scss';
+import {useAppSelector} from '@cdo/apps/util/reduxHooks';
+import Message from './Message';
+
+const AITutorChatWorkspace: React.FunctionComponent = () => {
```

Comment:
None of the existing logic in this component changed. I just removed unused logic and unnecessary divs/styling. I renamed these files and restructured them so that the organization of `AITutorContainer`, `AITutorChatWorkspace`, and `AITutorFooter` matches the structure of `RubricContainer`, `RubricContent`, and `RubricSubmitFooter` (the AI TA components). This allowed me to snag their most recent styling :)

Reply:
Just in case anyone is curious: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/rubrics/RubricContainer.jsx